lib/levanter/src/levanter/models/qwen.py

```diff
@@ -61,6 +62,10 @@ def hf_checkpoint_converter(
     def from_hf_config(cls, hf_config: HfConfig):
         rope_theta = hf_config.rope_theta
         rope_config = RotaryEmbeddingsConfig.from_hf_config(rope_theta, hf_config.rope_scaling)
+        use_bias = getattr(hf_config, "attention_bias", None)
+        if use_bias is None:
+            # Qwen2Config in newer transformers drops no_bias; assume bias by default
```

Comment:
Is this true? 

Qwen 2.5 doesn't specify any bias: https://huggingface.co/Qwen/Qwen2.5-7B/blob/main/config.json

but the model doesn't have bias. 
<img width="567" height="83" alt="Image" src="https://github.com/user-attachments/assets/70acea2d-768f-4df9-9f8d-fa5d0d0267d3" />

Pretty sure default False is correct (e.g. Qwen 2.5 doesn't work when I run this branch on a new eval because this is misconfigured)
```
KeyError: 'model.layers.mlp.gate_proj.bias'[32m [repeated 39x across cluster
```